### ISSUE TITLE
libicu48 is out of date

### ISSUE THREAD

**Issue by asimpson** [spfaffly/phantomjs-linux-armv6l#1]
Running `phantomjs -v` results in this error:

`error while loading shared libraries: libicudata.so.48: cannot open shared object file: No such file or directory`

apt-get is installing `libicu52`on my Raspian Jessie image. 

Is there a way to get v48? Or will you need to recompile?

**Comment by tdingman** [comment]
Bump. Just ran into the same error.

**Comment by dampen** [comment]
Bump. Same problem here.

**Comment by vanreece** [comment]
I ran into this problem when updating my raspberry pi to Jessie. However, as of that distro, I was able to install phantom js with a simple `sudo apt-get install phantomjs`. Worth trying out for you guys too?

**Comment by vanreece** [comment]
Not entirely sure that that one works though.... Starting the days long compilation process... :)

**Comment by sebpiq** [comment]
yes same problem here ... @vanreece when you're done compiling, would you mind maybe sharing your binaries? Or even maybe issuing a pull request :)

**Comment by vanreece** [comment]
Sure. It actually only took a couple hours on my new pi2! I'll issue a pull request.

**Comment by sebpiq** [comment]
great :) I have old pies ... so it's very slow here

**Comment by vanreece** [comment]
I don't have the time to do a whole github flow (and I'm on the wrong computer now), but let me know if this works for you.
[phantomjs-2.1.1-development-linux-armv7l.tar.gz](https://github.com/spfaffly/phantomjs-linux-armv6l/files/188330/phantomjs-2.1.1-development-linux-armv7l.tar.gz)

**Comment by sebpiq** [comment]
cool! Works with this build thanks :)

**Comment by vanreece** [comment]
Excellent. Glad to be of help! @asimpson @tdingman @dampen did you all find some other way to do this? If not try out that build I posted above.

**Comment by yunqu** [comment]
@vanreece Thanks for the sharing of the binaries. On arm-v7l, when I run `./phantomjs --version`, I see

```
 ./phantomjs: error while loading shared libraries: libicui18n.so.52: cannot open shared object file: No such file or directory
```
